Dockerfile

```diff
@@ -1,11 +1,11 @@
-FROM alpine:3.12.3
+FROM alpine:3.16.0
 LABEL "name"="sh-checker"
 LABEL "maintainer"="Luiz Muller <contact@luizm.dev>"
 
-ARG shfmt_version=3.4.3
+ARG shfmt_version=3.5.0
```

Comment:
Latest version is 3.5.1. See [here](https://github.com/mvdan/sh/releases/).

Reply:
Fixed...